assets/css/src/components/formEditor/components/_sidebar.scss

```diff
@@ -224,4 +224,34 @@ $form-placement-option-oval-border: #969ca1;
   display: grid;
   grid-row-gap: 20px;
   grid-template-columns: 1fr 45px;
+  margin-bottom: 40px;
+}
+
+.components-button.mailpoet-save-button {
+  background-color: #ff5301;
```

Comment:
I did it for most of the colours, not this one though. We will refactor this button when we do the big redesign, this will be some primary colour or something.